release-notes/opensearch-flow-framework.release-notes-2.15.0.0.md

```diff
@@ -0,0 +1,12 @@
+## 2024-06-11 Version 2.15.0.0
```

Comment:
Approving, but my lesson learned from last release is to just leave off this date.